flowauth/frontend/package-lock.json

```diff
@@ -221,7 +221,7 @@
     },
     "ansi-escapes": {
       "version": "3.1.0",
-      "resolved": "https://registry.npmjs.org/ansi-escapes/-/ansi-escapes-3.1.0.tgz",
+      "resolved": "http://registry.npmjs.org/ansi-escapes/-/ansi-escapes-3.1.0.tgz",
```

Comment:
Is there a reason that these URLs are not using https any more? Given several recent incidents in the Javascript world I'd really prefer these to use secure connections for package downloads if at all possible. Will things break if we switch this back?

Reply:
Seems to be this: https://github.com/npm/npm/issues/20719

Which the NPM people at least think isn't worth worrying about.

Reply:
Hmm, how is it _not_ worth worrying about someone potentially feeding you compromised versions of npm packages via a man-in-the-middle attack? 🤔

Again, if the installation still works with the https versions of the URLs I suggest simply reverting this change and not including it in the PR.

Reply:
Oh hang on, I may have mis-read what these links represent... What the "resolved" links actually used for?

Reply:
This is a pretty good explainer: https://github.com/npm/npm/issues/16849#issuecomment-312442508 but the short version would seem to be that these links aren't actually used to download the packages, and the protection is afforded via the integrity hash.

Fwiw, there are ~70 http:// links in the lock file before this as well ;)

Reply:
Fair enough. ;) Ok, with this background information I'm ok to leave it in as this is not security relevant (although still slightly annoying).